docs/src/public/userguide/_data/sidebars/netcdfJavaTutorial_sidebar.yml

```diff
@@ -194,10 +194,6 @@ entries:
       url: /cdm_netcdf_mapping.html
       output: web, pdf
 
-    - title: CDM Utility Programs
```

Comment:
We want to keep this one. There is an error in the source file that's resulting in the 404. It's a pretty easy fix, though. If you add these lines back, and remove the empty space at the beginning of the file `docs/src/public/userguide/pages/netcdfJava/developer/CdmUtilityPrograms.md`, then it will work.